### PR TITLE
Feat interpreter

### PR DESCRIPTION
**Pull request by Nitepone**

### CODE REVIEW

File: reader.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+:docformat: reStructuredText
+"""
+
+def main():
+    filepath = input("Filepath of Perkins Script: ")
+    readScript(filepath)
+
+
+def lineParser(line):
```
Comment (bitoffdev) [inline reader.py:15]: Should we consider using `str.split()`?

**Comment by bitoffdev** [comment]
So, the intent here is for this interpreter script to both parse the `.pbs` file and to make the calls to the controller?

File: reader.py (hunk 1 above)
Comment (Nitepone) [reply reader.py:15]: This method is used to allow us to accept string arguments in the future if needed.

**Review by Nitepone — COMMENTED** [review]
Yes, this interpreter will be responsible for interpreting the commands from `.pbs` files and making the appropriate calls from the controller. The controller will contain all functions to create patterns or otherwise manipulate the LEDs.

**Review by bitoffdev — APPROVED** [review]

**Pull request merged**